Classes/Controller/AbstractController.php

```diff
@@ -43,6 +46,16 @@ public function injectExtConf(ExtConf $extConf): void
         $this->extConf = $extConf;
     }
 
+    public function getQueryBuilderForTable(string $table)
+    {
+        return $this->getConnectionPool()->getQueryBuilderForTable($table);
+    }
+
+    public function getConnectionPool()
```

Comment:
return type ConnectionPool

Reply:
Nope. I have removed that method completely